html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <div>Price: <del>$19.99</del> $14.99</div>
+    <br>
```

Comment:
you want to avoid using the `br` tags here, the new lines are already created by the block elements